installer.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Clone the certification tools repo to a local directory.
+# If the repo is already available locally, fetch the latest version.
+# Use the --branch option to specify a specific branch
+
+
+fetch() {
+    if [ "$#" -lt 1 ]; then
+        echo "Usage: ${FUNCNAME[0]} <path> [<branch>]"
+        echo "Error: You need to provide a path to a local git repo."
```

Comment:
Does path really need to be provided or can we just have a safe default for it to use the repo name or "tools" or something? I like that you can specify a branch, but I suspect just sourcing this with no params would be sufficient in most cases.